include/eve/module/core/regular/impl/simd/arm/sve/first_true.hpp

```diff
@@ -20,44 +20,50 @@ namespace eve::detail
     auto cx = opts[condition_key];
 
     if constexpr (C::is_complete && !C::is_inverted) return std::nullopt;
+    else if constexpr (has_aggregated_abi_v<L>)
+    {
+      if constexpr (!C::is_complete)
+      {
+        if constexpr (relative_conditional_expr<C>) m = m && sve_true(cx, as(m));
+        else                                        m = m && expand_mask(cx, as<L>{});
+      }
```

Comment:
I think we should teach expand mask for relative conditional to call sve_true. Ottherwise this is annoying